unidist/core/backends/mpi/core/monitor/loop.py

```diff
@@ -170,6 +170,7 @@ def monitor_loop():
     The loop exits on special cancelation operation.
     ``unidist.core.backends.mpi.core.common.Operations`` defines a set of supported operations.
     """
+    monitor_logger.debug("Monitor loop started")
```

Comment:
```suggestion
```

Reply:
Added this looging line here so the init() function in root ends only after the monitor.log file is created and first log is written,  Or this file creation step though not very time consuming happens after init() of the root process and could make the first put() slightly slower as the monitor will be ready to recv operations only after creating the logger file.

Reply:
Discussed offline and decided to apply suggestion.

Aside: Is the log file created on the first write or when we call get_logger?

Reply:
ping

Reply:
Verified again the log file is created when the "first write" is made.

Reply:
Thanks!